libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
`getCurrentFramebufferId()` is available in both GL renderers, yes.

Ideally, i'd like to get rid of the getRenderer() calls as much as possible, too. 

But here, we need a way to query the renderer state, since only the renderer keeps track of & knows the currently bound FBO...